components/Cooperation/List.tsx

```diff
@@ -0,0 +1,67 @@
+import { Cooperation } from '@open-source-bazaar/activityhub-service';
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import { ObservedComponent } from 'mobx-react-helper';
+import { Column, RestTable } from 'mobx-restful-table';
+import { Container } from 'react-bootstrap';
+
+import { CooperationModel } from '../../models/Cooperation';
+import { i18n, I18nContext } from '../../models/Translation';
+import { PageHead } from '../Navigator/PageHead';
+
+export interface CooperationListProps {
+  activityId: number;
+}
+
+@observer
+export class CooperationList extends ObservedComponent<CooperationListProps, typeof i18n> {
+  static contextType = I18nContext;
+
+  cooperationStore = new CooperationModel(this.props.activityId);
+
+  @computed
+  get columns(): Column<Cooperation>[] {
+    const { t } = this.observedContext,
+      { activityId } = this.observedProps;
+
+    return [
+      {
+        key: 'partner',
+        renderHead: t('organization'),
+        renderBody: ({ id, partner }) => (
+          <a href={`/activity/${activityId}/cooperation/${id}/editor`}>
+            {partner?.name || t('unknown')}
```

Comment:
```suggestion
          <a href={`/organization/${id}`}>
            {partner?.name || t('unknown')}
```
`<RestTable />`有专门的 `editable` 属性提供编辑功能，只需定义好每个字段的输入控件属性，即可自动弹框编辑，无需单独的编辑页。